docker_install.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+
+apt-get update && apt-get install -y apt-transport-https ca-certificates curl software-properties-common
+curl -fsSL https://mirrors.aliyun.com/docker-ce/linux/ubuntu/gpg | apt-key add -
+add-apt-repository 'deb [arch=amd64] https://mirrors.ustc.edu.cn/docker-ce/linux/ubuntu bionic stable'
```

Comment:
We can directly use official docker install guide to reduce maintenance costs. https://docs.docker.com/engine/install/debian/.